files/ko/web/api/web_storage_api/index.md

```diff
@@ -49,12 +49,6 @@ Web Storage의 두 가지 방식은 다음과 같습니다.
 
 ## 브라우저 호환성
 
-### `Window.localStorage`
-
-{{Compat}}
-
-### `Window.sessionStorage`
```

Comment:
Previously, just did `{{Compat}}` conversion, but I didn't expect this to happen.